src/applications/mhv/landing-page/components/LandingPage.jsx

```diff
@@ -3,23 +3,36 @@ import { useSelector } from 'react-redux';
 import PropTypes from 'prop-types';
 import CardLayout from './CardLayout';
 import NoHealthAlert from './NoHealthAlert';
+import HeaderLayoutV1 from './HeaderLayoutV1';
 import HeaderLayout from './HeaderLayout';
 import HubLinks from './HubLinks';
 import NewsletterSignup from './NewsletterSignup';
-import { hasHealthData } from '../selectors';
+import Welcome from './Welcome';
+import {
+  hasHealthData,
+  personalizationEnabled,
+  selectGreetingName,
+} from '../selectors';
 
-const LandingPage = ({ data = null }) => {
-  const { cards = null, hubs } = data;
-  // Show the cards only for those users with health data.
+const LandingPage = ({ data = {} }) => {
+  const { cards = [], hubs = [] } = data;
+  const name = useSelector(selectGreetingName);
   const showCards = useSelector(hasHealthData);
+  const showPersonalization = useSelector(personalizationEnabled);
 
   return (
     <div
       className="vads-u-margin-y--3 medium-screen:vads-u-margin-y--5"
       data-testid="landing-page-container"
     >
       <div className="vads-l-grid-container large-screen:vads-u-padding-x--0">
-        <HeaderLayout />
+        {!showPersonalization && <HeaderLayoutV1 />}
```

Comment:
I think this is fine as is. It may be possible to condense the two conditionals `showPersonalization` and `showPersonalization` as the following:

````
{showPersonalization ? (
          <>
            <HeaderLayout />
            <Welcome name={name} />
          </>
        ) : (
          <HeaderLayoutV1 />
        )}
````

Reply:
I was trying not to insert a ternary into JSX, as I feel it is less ledgible than including two conditional rendering statements.

Either is fine to me, personally. If someone else wants to weigh in, I'd be happy to change it (or leave it).

Reply:
I say just leave it as it, I agree with the readability being better without a ternary.